apps/test-bundles/package.json

```diff
@@ -33,6 +33,8 @@
   },
   "dependencies": {
     "@uifabric/set-version": ">=1.1.3 <2.0.0",
+    "@uifabric/styling": ">=6.41.0 <7.0.0",
+    "office-ui-fabric-react": ">=6.141.0 <7.0.0",
```

Comment:
This is duplicated a couple lines below?

Reply:
resolved.